### PR TITLE
Enable direct `FieldTimeSeries` forcing

### PR DESCRIPTION
**Pull request by simone-silvestri**
this PR enables using `FieldTimeSeries` for direct forcing as
```julia
u_forcing = FieldTimeSeries{}(...)
forcing = Forcing(u_forcing)
model = NonhydrostaticModel(; grid, forcing = (; u = u_forcing))
```

### CODE REVIEW

**Comment by glwagner** [comment]
What's the point of wrapping it in `Forcing`?

I think a cleaner syntax is 

```julia
u_forcing = FieldTimeSeries{}(...)
model = NonhydrostaticModel(; grid, forcing = (; u = u_forcing))
```

**Comment by simone-silvestri** [comment]
yeah sorry, I meant that. I don't know why I slipped that extra `forcing = Forcing(u_forcing)`

File: src/Forcings/model_forcing.jl [hunk 1]
```diff
@@ -27,6 +27,8 @@ regularize_forcing(::Nothing, field::AbstractField, field_name, model_field_name
 
 # TODO: some checking that `array` is validly-sized could be done here
 regularize_forcing(array::AbstractArray, field::AbstractField, field_name, model_field_names) = Forcing(array)
+regularize_forcing(fts::FlavorOfFTS, field::AbstractField, field_name, model_field_names) = Forcing(fts)
```
Comment (glwagner) [inline src/Forcings/model_forcing.jl:30]: the same TODO above applies to FTS too

Comment (glwagner) [reply src/Forcings/model_forcing.jl:30]: not just size but for FTS we can also check location and even the grid

**Review by glwagner — APPROVED** [review]

**Pull request merged**